src/scripts.js

```diff
@@ -110,9 +94,9 @@ const setTripsDestination = () => {
 
 const findYearlySpent = (travelersTrips) => {
 let yearlySpent = travelersTrips.reduce((acc, trip) => {
-    if(trip.date.includes('2022')) {
+    if(dayjs(trip.date).year() === dayjs().year()) {
```

Comment:
Remember, this isn't solving for what the project is asking for. The project asks for the total value over the last year not during the current year.

What you need to do here is create a dayjs date using subtract 1 "year" and filter out all trips that have a dayjs(trip.date) that is greater than a year ago (i.e. within the last year).